### PR TITLE
Mark the latest v1.19.x release as stable

### PR DESCRIPTION
**Pull request by brandond**
#### Proposed Changes ####

Mark the latest v1.19.x release as stable - final task from 2020-10-30 release checklist

#### Types of Changes ####

* Channel

#### Verification ####

* Install k3s from latest channel
* Check version with k3s --version

#### Linked Issues ####

Related to #2460

#### Further Comments ####

### CODE REVIEW

**Comment by davidnuzik** [comment]
We can bump 1.18 but I thought we were not quite ready to mark v1.19.x as stable based on some recent conversations around SELinux? I thought we also wanted some more users to test out etcd.

File: channel.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Example channels config
 channels:
 - name: stable
-  latest: v1.18.9+k3s1
+  latest: v1.19.3+k3s2
```
Comment (davidnuzik) [inline channel.yaml:4]:
```suggestion
  latest: v1.18.10+k3s2
```

**Review by davidnuzik — CHANGES_REQUESTED** [review]
We can bump 1.18 but I thought we were not quite ready to mark v1.19.x as stable based on some recent conversations around SELinux? I thought we also wanted some more users to test out etcd.

**Comment by brandond** [comment]
Was it 1.19 that we're not comfortable with, or just not ready to call etcd stable yet?  I can do either. @cjellick ?

**Comment by davidnuzik** [comment]
Per our discussion today in Standup, we're okay with marking `v1.19.3+k3s2` as stable.

File: channel.yaml (hunk 1 above)
Comment (davidnuzik) [reply channel.yaml:4]: Per 11/9/20 standup convo, we are okay with v1.19.3+k3s2

**Review by davidnuzik — APPROVED** [review]
LGTM

**Pull request merged**